YAXLib/StringUtils.cs

```diff
@@ -255,10 +255,10 @@ public static bool IsSingleLocationGeneric(string location)
         }
 
         /// <summary>
-        ///     Gets the string corresponidng to the given array dimensions.
+        ///     Gets the string corresponiding to the given array dimensions.
         /// </summary>
         /// <param name="dims">The array dimensions.</param>
-        /// <returns>the string corresponidng to the given array dimensions</returns>
```

Comment:
I like the old wording 


....No just joking :p

Reply:
Me, too😆